airbyte-integrations/connectors/source-trello/pyproject.toml

```
@@ -0,0 +1,28 @@
+[build-system]
+requires = [ "poetry-core>=1.0.0",]
+build-backend = "poetry.core.masonry.api"
+
+[tool.poetry]
+version = "1.0.3"
+name = "source-trello"
+description = "Source implementation for Trello."
+authors = [ "Airbyte <contact@airbyte.io>",]
+license = "MIT"
+readme = "README.md"
+documentation = "https://docs.airbyte.com/integrations/sources/trello"
+homepage = "https://airbyte.com"
+repository = "https://github.com/airbytehq/airbyte"
+[[tool.poetry.packages]]
+include = "source_trello"
+
+[tool.poetry.dependencies]
+python = "^3.9,<3.12"
+airbyte-cdk = "^0"
+
+[tool.poetry.scripts]
+source-trello = "source_trello.run:run"
+
+[tool.poetry.group.dev.dependencies]
+pytest = "^6.2"
```

```suggestion
pytest = "^8"
```

Might require a `poetry lock` to apply, this can be done in a separate PR